src/internal/operators/switchScan.ts

```diff
@@ -0,0 +1,51 @@
+/** @prettier */
+import { ObservableInput, ObservedValueOf, OperatorFunction } from '../types';
+import { switchMap } from './switchMap';
+import { operate } from '../util/lift';
+
+/**
+ * Applies an accumulator function over the source Observable where the
+ * accumulator function itself returns an Observable, emitting values
+ * only from the most recently returned Observable.
+ *
+ * <span class="informal">It's like {@link scan}, but only the most recent
+ * Observable returned by the accumulator is merged into the outer Observable.</span>
+ *
+ * ![](switchScan.png)
```

Comment:
question: @niklas-wortmann With the addition of a new operator, is there some config that needs to updated so that a marble diagram for it is generated?

Reply:
Added a TODO around this.